### ISSUE TITLE
[Feature]: Enable separate mappings for Bestiary and Monster Core Token packs 

### ISSUE THREAD

**Issue by pinkflumph** [foundryvtt/foundryvtt-premium-content#634]
### What is the feature?

**Requested feature**
Add a setting that would remove overlapping mappings for creatures that are featured in both the Bestiaries and Monster Core when both modules are installed

**Details**
As it stands, installing both modules results in art from one of them overriding art from the other for creatures featured in both. Many such creatures have different art in the two modules, so the overlap needlessly removes one of the two versions, resulting in less variety and making it harder to distinguish between the two in the compendium browser. 

Having an option to remove the overlapping mapping would allow one to keep using Bestiary token art for creatures from the Bestiaries, while using Monster Core token art for creatures from Monster Core   

### What package are you reporting this for?

Pathfinder Token Pack: Bestiaries

**Comment by racehd** [comment]
Core bug: https://github.com/foundryvtt/foundryvtt/issues/11591

If a temporary fix is desired, a regex could potentially capture missing scales and insert them. I began testing this and have a somewhat working test pattern.

**Comment by pinkflumph** [comment]
@racehd, the feature I am requesting here is not related to the scale issue. Whether the mapping is merged or overridden, it remains true that Bestiaries and Monster Core have some overlapping mappings, only one of each can apply at a time, resulting in one of the two modules' art being unused 

This isn't a bug by the way, it's a feature - the mappings are done that way so that Bestiary creatures duplicated in Monster Core get art in both of their versions even if you have just one module installed. Unfortunately, it results in unused art when both are installed 

The solution I have currently implemented is to create a very simple Foundry module which adds a mapping file with just the Monster Core mappings (or Bestiary mappings, either would work), and then to disable the built-in Monster Core (Bestiary) mappings in settings. This way Bestiary creatures use Bestiary art and Monster Core creatures use Monster Core art

**Comment by silvative** [comment]
> I see, my apologies. One possible solution-- instead of the mapping-pf2e.json being one file, it could be split into its primary target (bestiaries) and secondary target (monster core creatures that have bestiary art). Then in core settings you could set your preference as:
> 
>     1. Monster Core
> 
>     2. Bestiary
> 
>     3. Monster Core creatures that exist in Bestiary
> 
>     4. Bestiary creatures that exist in Monster Core
> 
> 
> or vice versa. This would leverage the existing core functionality and files, just split up.
> 
> Edit: though it appears the core 'configure art' option lists by module, not by mapping. So my solution would also need a core enhancement to separate out. another solution would be to split the extra mapping into a free module which could be enabled by users who own bestiary but not monster core.

I agree that this would be an ideal solution, but unfortunately several mapping files isn't something that is supported by the image replacement framework at this time, so it would require a change in the core software :c
